operations/helm/charts/mimir-distributed/values.yaml

```diff
@@ -86,6 +86,9 @@ externalConfigSecretName: '{{ include "mimir.resourceName" (dict "ctx" . "compon
 # -- When 'useExternalConfig' is true, then changing 'externalConfigVersion' triggers restart of services - otherwise changes to the configuration cause a restart.
 externalConfigVersion: '0'
 
+# -- Enables Vault Agent to mount secrets to TLS configurable components. This requires Vault and Vault Agent to already be running.
+useVaultAgent: false
```

Comment:
If the vault agent is configurable per component basis, I am thinking maybe we should just put the config in each component level, instead of a one-flag-catch-all like this. 

I.e:

```
compactor:
  vaultAgentEnabled: false
ingester:
  vaultAgentEnabled: false
# and so on
```